### DISCUSSION TITLE
Made a plugin for `asdf` (asdf-vm.com)

### DISCUSSION THREAD

**Discussion by donaldguy** [bensadeh/tailspin#115] [Show and tell]
It's here: https://github.com/donaldguy/asdf-tspin

See: https://asdf-vm.com if you are unfamiliar with the general project, but the TL;DR is people should be add to install pinned version as

```shell
# install asdf, as a git clone or with package manager
asdf plugin add tspin https://github.com/donaldguy/asdf-tspin.git
asdf install tspin latest

asdf shell tspin 3.0.0

# or the probs more common
asdf local tspin 3.0.0
# which writes into a repo-local `.tool-versions` and shims the loads
```

(This only took a few min; 
I used the https://github.com/asdf-vm/asdf-plugin-template 
and then took the platform/toolchain logic from https://github.com/ggilmore/asdf-just/blob/9e8e82cb232b231675873c0157aa4141e9fbe012/lib/utils.bash#L37)

potentially also should work with e.g. https://github.com/jdx/mise but haven't tested.

Anyway ... FYI

If you want this to be an "official" plugin (so the first bit would just be `asdf plugin add tspin`, rather than requiring the URL), it would be a PR to https://github.com/asdf-vm/asdf-plugins 

Lmk if you are interested broadly (and e.g. if you want me to e.g. transfer you the repo first)

**Comment by bensadeh** [comment]
Sorry, looks like I missed this! Didn't know you could do that with `asdf`!

Thanks for taking the time to put this together. For now, I'm currently focusing on other projects and commitments, so I won't be able to take on the responsibility of maintaining another plugin or working towards making it an official part of the asdf plugins.

That being said, I'm very flattered that people are interested in packaging and offering tailspin in so many ways, so thanks again for that 🙏 .
